production/notes.md

```diff
@@ -0,0 +1,20 @@
+create user, add to sudoers, install ssh key
```

Comment:
That would be good. I was just planning to create an image of the droplet
for expediency's sake. But if you want to create a setup script feel free!

On Monday, December 14, 2015, Ryan Brown <notifications@github.com> wrote:

> In production/notes.md
> <https://github.com/mediapublic/mediapublic/pull/151#discussion_r47497190>
> :
>
> > @@ -0,0 +1,20 @@
> > +create user, add to sudoers, install ssh key
>
> Can we maybe make this a script that actually runs as root?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mediapublic/mediapublic/pull/151/files#r47497190>.
>